diary-alexander-jaffray/modernized/37-concluding-observations.adoc

```diff
@@ -1,9 +1,7 @@
 == Concluding Observations
 
-On the state of the church at large,
-with reference to the successive stages of her reformation,
-and occasioned by a view of that share and interest,
-which the Society of Friends have ever taken therein.
+[.chapter-synopsis]
+* On the state of the church at large, with reference to the successive stages of her reformation, and occasioned by a view of that share and interest, which the Society of Friends have ever taken therein
```

Comment:
This chapter-synopsis looks good. But looks like you forgot to do it on the original too. :)